internal/matcher/match.go

```diff
@@ -112,10 +113,7 @@ func EnrichedMatch(ctx context.Context, ir *claircore.IndexReport, ms []driver.M
 				}
 				vs, err := NewController(m, s).Match(mctx, records)
 				if err != nil {
-					zlog.Error(ctx).
-						Err(err).
-						Msg("matcher error")
-					continue
+					return fmt.Errorf("matcher error: %v", err)
```

Comment:
```suggestion
					return fmt.Errorf("matcher error: %w", err)
```

Reply:
Yep you're right it should be %w, thanks! Will update and rebase as well.

Interestingly the checkbox was already ticked when I found it :/ 
![Screenshot 2022-05-17 at 12 57 42](https://user-images.githubusercontent.com/28044847/168805710-cba5dd5d-4d71-44d3-9998-394391fe1e28.png)